docs/docs/models/ESM-2/pre-training.md

```diff
@@ -10,7 +10,7 @@ and train/test splits are available.
 Validation perplexity evaluated on the NVIDIA validation set.
 
 <figure markdown="span">
```

Comment:
when you go to the file, some images are not loaded

Reply:
there is also `=== "8M"`

Reply:
The `===` is how material for mkdocs does content tabs: https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#grouping-other-content

But I'm not sure the images will load in the github markdown renderer; this is mainly intended to get rendered as part of the docs pages.

Reply:
hmm, maybe it's because they're SVGs? I could try with a PNG and see if that renders

Reply:
Ok, added these as PNGs and they seem to be rendering OK now.